common/shlibs

```diff
@@ -641,10 +641,10 @@ libip6tc.so.2 iptables-1.8.3_1
 libxtables.so.12 iptables-1.6.1_1
 libip4tc.so.2 iptables-1.8.3_1
 libipq.so.0 iptables-1.4.7_1
-libnm-util.so.2 libnm-0.9.1.90_1
-libnm-glib.so.4 libnm-0.9.1.90_1
-libnm-glib-vpn.so.1 libnm-0.9.1.90_1
-libnm.so.0 libnm-1.0.0_1
+libnm-util.so.2 libnm-1.18.2_1
+libnm-glib.so.4 libnm-1.18.2_1
+libnm-glib-vpn.so.1 libnm-1.18.2_1
+libnm.so.0 libnm-1.18.2_1
```

Comment:
Is there a reason that this was updated?

Reply:
Not yet. Forgot to add WIP.

Reply:
The reason to update was this: https://github.com/void-linux/void-packages/pull/13538#issuecomment-519085008
But it did not work.